DUI3-DX/DUI3/Speckle.Connectors.DUI/Bridge/BrowserBridge.cs

```diff
@@ -223,7 +224,7 @@ private void ExecuteMethod(string methodName, string requestId, string args)
       else // It's an async call
       {
         // See note at start of function. Do not asyncify!
-        resultTypedTask.Wait();
+        resultTypedTask.Wait(); // Throws Aggregate exception
```

Comment:
If we'd prefer, `.GetAwaiter().GetResult()` will not wrap the exception in an Aggregate exception.

Reply:
I'll test this!

Reply:
You are right! It simplified things too